mad-dog/src/index.js

```diff
@@ -132,7 +166,24 @@ async function generateLibsInstances (numUsers, useZeroIndexedWallet = false) {
 // with a separate command.
 async function main () {
   logger.info('🐶 * Woof Woof * Welcome to Mad-Dog 🐶')
+
+  logger.info('Ensuring all nodes are healthy..')
+  try {
+    await Promise.all(
```

Comment:
isn't this handled in existing setup already?

Reply:
I don't think so. If you run `npm run start <test name>`, it runs the test without performing any health checks

Reply:
aren't all services health checked when being brought up? as long as health checks are reliably handled in CI, it seems perfectly reasonable to allow test runners to manually bypass health check

Reply:
I mentioned this in a previous comment where in local development, if a service is in hot-reload mode, the service may not be ready yet. we can disable this health check in CI or just keep it all together